src/test/java/uk/gov/hmcts/reform/amapi/controllers/AccessManagementControllerTest.java

```diff
@@ -100,7 +100,7 @@ public void testRevokeResourceAccess() throws Exception {
 
         doNothing().when(accessManagementService).revokeResourceAccess(any());
 
-        this.mockMvc.perform(delete("/lib/revoke-resource-access")
+        this.mockMvc.perform(delete("/api/revoke-resource-access")
```

Comment:
Change the rest api end point to Delete /access-resource

Reply:
done